Infrastructure/src/graphics/textengine.cpp

```diff
@@ -463,6 +463,25 @@ namespace gfx {
 		mImpl->textFormats.clear();
 	}
 
+	bool TextEngine::HasFontFamily(const std::string & name)
```

Comment:
ya never ended up needing this :)

In the way you refactored the code below, TextLayouter::Measure() does the checking (though via the fonts/mapping.json from TioVfs, not via querying the FontCollection object).

Reply:
Heh yeah, true. I left it in in case we do need it at some point because the functionality didnt exist before.